quotas-and-limits_5cc850ba2c7d3a026fd42e7e.md

```diff
@@ -0,0 +1,81 @@
+Semaphore 2.0 enforces usage quotas and limits to protect customers and the
+platform from unforeseen spikes in usage. As your use of Semaphore expands over
+time, your quotas may increase accordingly.
+
+If you expect a notable upcoming increase in usage, you can proactively request
+quota adjustments by sending a request to <support@semaphoreci.com>.
+
+- [Quotas for Maximum Concurrent Running Jobs in an Organization](#quotas-for-maximum-concurrent-running-jobs-in-an-organization)
+- [Pipeline and Block Execution Time Limit](#pipeline-execution-time-limit)
+- [Job Log Output Size Limit](#job-log-output-size-limit)
+
+## Quotas for Maximum Concurrent Running Jobs in an Organization
+
+Every organization has a set of quotas that dictate the maximum number of
+concurrently running jobs.
+
+Default quotas per machine type for a newly created organization:
+
+| Machine Type  | Default Quota |
+| ------------- | ------------- |
+| e1-standard-2 | 8             |
+| e1-standard-4 | 8             |
+| e1-standard-8 | 8             |
+| a1-standard-4 | 2             |
+
+## Pipeline and Block Execution Time Limit
+
+By default, every pipeline and block is limited to one hour execution time. This
+mechanism protects your project from undesired expenses in case of jobs that are
+running longer than anticipated. A good example is an accidentally committed
+debug statement that is waiting for user input.
+
+This limit is adjustable in the Pipeline YAML.
+
+``` yaml
+version: v1.0
+name: Using execution_time_limit
+
+agent:
+  machine:
+    type: e1-standard-2
+    os_image: ubuntu1804
+
+execution_time_limit:
+  hours: 3
+
+blocks:
+  - name: Limited to 3 hours by pipeline level configuration
+    commands:
+      - sudo apt-get install piglet
+
+  - name: Limited to 15 minutes
+    execution_time_limit:
+      minutes: 15
+    commands:
+      - sudo apt-get install vim
+```
+
+For detailed explanation, see the [execution_time_limit limit section in the
+Pipeline YAML reference][execution_time_limit_reference].
+
+## Job Log Output Size Limit
+
+Semaphore collects up to 16 megabytes of raw log data from every job in a
+pipeline, which in practice roughly equals 100000 lines of output.
+
+Logs longer than 16 megabytes are trimmed with the following message on the
+bottom:
+
+``` txt
+Content of the log is bigger than 16mb. Log is trimmed.
+```
+
+This limit is not adjustable.
+
+For collecting longer textual files, or output from long and verbose process,
+we recommend using a blob store like AWS S3 or Google Cloud Storage. If you have
+a specific use case for longer logs available as part of the job log output,
```

Comment:
This last sentence is not compatible with _This limit is not adjustable_.

Reply:
Good point. Removed the last sentence.